alarm/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id("configure-npm")
+}
+
+packageJson {
+    name = "alarm"
+    description = "Alarm microservice"
+    main = "dist/src/index.js"
+    scripts {
+        val scriptDeps = listOf(npmScript("build") inProject "common")
+        listOf(
+            "test" runs "vitest --exclude \"test/tolerance/**\" --run",
```

Comment:
i think we can remove the --exclude ...

Reply:
@paga16-hash it seems that the tolerance tests are not passing...